src/Controller/GalleryController.php

```diff
@@ -30,6 +33,51 @@ public function showImagesByCategorie(string $categorie, ImageRepository $imageR
         return $this->render('gallery/metallicDesign.html.twig', ['images' => $images]);
     }
 
+    /**
+     * @Route("/gallerie-design-metallique/{categorie}/{id}",
+     *          name="gallery_metallicDesign_category_showImage", methods={"GET"})
+     */
+    public function showImage(Image $image, string $categorie): Response
+    {
+        return $this->render('gallery/showImage.html.twig', [
+            'categorie' => $categorie,
```

Comment:
Isn't the category an image property ?
What's the point of having it in the URL to send it directly to the view instead of just getting the image's category ?

Reply:
make changes